core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -61,11 +61,17 @@
  */
 public final class InProcessTelemetryChannel implements TelemetryChannel {
     private final static int DEFAULT_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 500;
+    private final static int MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 1;
+    private final static int MAX_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 500;
 
     private final static int TRANSMIT_BUFFER_DEFAULT_TIMEOUT_IN_SECONDS = 10;
+    private final static int MIN_TRANSMIT_BUFFER_TIMEOUT_IN_SECONDS = 1;
+    private final static int MAX_TRANSMIT_BUFFER_TIMEOUT_IN_SECONDS = 30;
 
     private final static String DEVELOPER_MODE = "DeveloperMode";
-    private final static String EndpointAddress = "EndpointAddress";
+    private final static String ENDPOINT_ADDRESS = "EndpointAddress";
+    private final static String MAX_TELEMETRY_ITEMS_IN_QUEUE = "MaxTelemetryItemsInQueue";
```

Comment:
These new two configuration elements should be aligned with the .Net SDK.

Also queue is an internal implementation details so it might be better to use MaxTelemetryItemsBatchSize or something similar